src/components/Layout/PageHeader.vue

```diff
@@ -5,8 +5,12 @@
                 <div class="ping-section">
                     <p>[Demo build]</p>
                     <p>Ping:</p><p v-if="!ping">waiting</p><p>{{ ping }}</p>
-                    <p>Username:</p><p v-if="user">{{user.username}}</p>
-                    <p>Group:</p><p v-if="studentGroup">{{ studentGroup.name }}</p>
+                    <div v-if="user">
+                        <p>Username:</p><p v-if="user">{{ user.username }}</p>
```

Comment:
You forgot to remove the `v-if` - since we're inside the div, we already know the `user` exists.